crates/holochain/src/conductor/conductor.rs

```diff
@@ -1319,6 +1320,7 @@ mod app_impls {
 
     impl Conductor {
         #[cfg(feature = "test_utils")]
+        // FIXME: can rewrite now in terms of `get_install_app_payload_from_dnas`
```

Comment:
Something open in v1 to track this?

Reply:
@abe-njama could you make a story for this for some time in the future? "Rewrite install_app_legacy in terms of get_install_app_payload_from_dnas"